README.md

```diff
@@ -113,6 +113,7 @@
 * [Roadmap](#roadmap)
 * [Editor Support](#editor-support)
 * [Contributing](.github/CONTRIBUTING.md)
+* [Development](#development)
```

Comment:
this is a generated file. you can look at the doc folder, and you will see a sub folder with templates for the nav and content md files.
so this goes to the nav.
also, we should not add a new section, instead lets use the Editor Support section that already exists and talks about vim.